pkg/replica/replica.go

```diff
@@ -772,11 +772,15 @@ func (r *Replica) linkDisk(oldname, newname string) error {
 	}
 
 	dest := r.diskPath(newname)
-	if _, err := os.Stat(dest); err == nil {
-		logrus.Infof("Old file %s exists, deleting", dest)
-		if err := os.Remove(dest); err != nil {
-			return err
-		}
+	if err := os.RemoveAll(dest); err != nil {
+		logrus.Infof("Deleting old disk file %v", dest)
```

Comment:
This only shows on error, should we move this out of the if statement?

Reply:
Updated.  Thanks.